README.md

```diff
@@ -4,44 +4,27 @@
 [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 # Project Calico
 
-Project Calico provides
-
-- A simple, pure layer 3 networking approach with no overlays for networking
-  "workloads" such as VMs and containers.
-- A distributed firewall implementing rich and flexible network policy,
-  imposed at ingress/egress to each workload.
-
-For more information see [the Project Calico website](http://www.projectcalico.org/learn/).
+<blockquote>
```

Comment:
Certainly supported and works.  Happy to merge if that's the way you'd prefer, but I much prefer the formatting of the way I suggested (both the markdown and the final rendered result).

Doesn't really matter though.